.github/workflows/spell.yml

```diff
@@ -0,0 +1,19 @@
+name: spell
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  spell:
+    name: Check spelling
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Can you lock both actions to given SHA with tag comment like it is done at

https://github.com/rubygems/rubygems/blob/75ec11612de0818c1efb83db017484c65b2c8ea4/.github/workflows/daily-bundler.yml#L23

:question:

Reply:
Sure thing. Done in the latest push.